homeassistant/components/switcher_kis/cover.py

```diff
@@ -44,37 +44,30 @@ def async_add_cover(coordinator: SwitcherDataUpdateCoordinator) -> None:
             DeviceCategory.SHUTTER,
             DeviceCategory.SINGLE_SHUTTER_DUAL_LIGHT,
         ):
-            async_add_entities([SwitcherCoverEntity(coordinator, 0)])
+            async_add_entities([SwitcherSingleCoverEntity(coordinator, 0)])
+        if (
+            coordinator.data.device_type.category
+            == DeviceCategory.DUAL_SHUTTER_SINGLE_LIGHT
+        ):
+            async_add_entities([SwitcherDualCoverEntity(coordinator, 0)])
+            async_add_entities([SwitcherDualCoverEntity(coordinator, 1)])
```

Comment:
We should not call `async_add_entities` more than once at a time. Please collect the entities in a list and add them in one go